package.json

```diff
@@ -50,6 +56,30 @@
     "react": ">=15"
   },
   "devDependencies": {
+    "@storybook/react": "^3.2.3",
+    "all-contributors-cli": "^4.3.0",
```

Comment:
I believe most of these dependencies are handled by kcd-scripts. @kentcdodds, can you verify which dependencies need to be added. For example, there are already storybook scripts in `package.json` so I wouldn't expect that dependency to need to be added here.